styles/contactinfo.css

```diff
@@ -0,0 +1,44 @@
+/*
+  Contact info styles
+*/
+
+
+.contactInfo__back {
+  color: rgb(255, 0, 0);
+  margin-top: 15px;
+  padding-left: 10px;
+  position: absolute;
+  text-decoration: none;
+  top: 0px;
```

Comment:
Two things
  1. The use of `px` is not needed with `0`
  2. Unless using _positioning_ (i.e. `position: absolute`) this will not accomplish anything worthwhile/explicit